arviz/plots/loopitplot.py

```diff
@@ -0,0 +1,208 @@
+    use_hpd : bool, optional
+        Use plot_hpd to fill between hpd values instead of overlaying the uniform distributions.
+    credible_interval : float, optional
+        Credible interval of the hpd or of the ECDF theoretical credible interval
+    figsize : figure size tuple, optional
+        If None, size is (8 + numvars, 8 + numvars)
+    textsize: int, optional
+        Text size for labels. If None it will be autoscaled based on figsize.
+    color : str or array_like, optional
+        Color of the LOO-PIT estimated pdf plot.
+    legend : bool, optional
+        Show the legend of the figure.
+    ax : axes, optional
+        Matplotlib axes
+    plot_kwargs : dict, optional
+        Additional keywords passed to ax.plot for LOO-PIT line (kde or ECDF)
+    plot_unif_kwargs : dict, optional
+        Additional keywords passed to ax.plot for overlaid uniform distributions or
+        for beta credible interval lines if ``ecdf=True``
+    hpd_kwargs : dict, optional
+        Additional keywords passed to az.plot_hpd
+    fill_kwargs : dict, optional
+        Additional kwargs passed to ax.fill_between
+
+    Returns
+    -------
+    axes : axes
+        Matplotlib axes
+
+    References
+    ----------
+    * Gabry et al. (2017) see https://arxiv.org/abs/1709.01449
+    * https://mc-stan.org/bayesplot/reference/PPC-loo.html
+    * Gelman et al. BDA (2014) Section 6.3
+
+    Examples
+    --------
+    Plot LOO-PIT predictive checks overlaying the KDE of the LOO-PIT values to several
+    realizations of uniform variable sampling with the same number of observations.
+
+    .. plot::
+        :context: close-figs
+
+        >>> import arviz as az
+        >>> idata = az.load_arviz_data("centered_eight")
+        >>> az.plot_loo_pit(idata=idata, y="obs")
+
+    Fill the area containing the 94% credible interval of the difference between uniform
+    variables empirical CDF and the real uniform CDF. A LOO-PIT ECDF clearly outside of these
+    theoretical boundaries indicates that the observations and the posterior predictive
+    samples do not follow the same distribution.
+
+    .. plot::
+        :context: close-figs
+
+        >>> az.plot_loo_pit(idata=idata, y="obs", ecdf=True)
+
+    """
+    if ecdf and use_hpd:
+        raise ValueError("use_hpd is incompatible with ecdf plot")
+
+    (figsize, _, _, xt_labelsize, linewidth, _) = _scale_fig_size(figsize, textsize, 1, 1)
+    if ax is None:
+        _, ax = plt.subplots(1, 1, figsize=figsize, constrained_layout=True)
+
+    if plot_kwargs is None:
+        plot_kwargs = {}
+    plot_kwargs["color"] = color
+    plot_kwargs.setdefault("linewidth", linewidth * 1.4)
+    plot_kwargs.setdefault("label", "LOO-PIT ECDF" if ecdf else "LOO-PIT")
+    plot_kwargs.setdefault("zorder", 5)
+
+    if plot_unif_kwargs is None:
+        plot_unif_kwargs = {}
+    light_color = rgb_to_hsv(to_rgb(plot_kwargs.get("color")))
```

Comment:
should we do this only if the user has not provided the color?

Reply:
Do you mean that the default when color is present should be that loo-pit line and unif lines have the same color? Or that when plot_unif_kwargs has the key color this calculation should be avoided?

Reply:
I decided on this behaviour so that loo-pit line and unif lines have similar but different color, unless the user sets color in plot_unif_kwargs. I think that they should be slightly different, which is why using exactly equal colors is not default nor _defaultish_

Reply:
Ok, now I understand, sounds good

Reply:
I will try to clarify this behaviour on the docs.